.github/workflows/continuous_integration.yml

```diff
@@ -47,27 +47,27 @@ jobs:
       - name: Test with tox
         run: tox
       - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v3
+        uses: codecov/codecov-action@v4
         with:
           file: coverage.xml
 
   linting:
     name: Linting
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/cache@v3
+      - uses: actions/checkout@v4
+      - uses: actions/cache@v4
```

Comment:
Someone recently added dependabot for these. We should be able to rely on that for github actions, but the rest of the version bumps look good. 

Mind removing the GH action updates so we can let Dependabot take care of it?

Reply:
done